README.md

```diff
@@ -1,11 +1,12 @@
 # coord-env-interface
-Interface definition between coordination algorithms and environments. Includes a dummy algorithm and environment as example.
+Interface definition between coordination algorithms and environments. Includes a dummy algorithm, Non-RL algorithms(Shortest Path, Load Balance) and environment as example.
```

Comment:
What does the dummy algo do? It's not an RL algorithm either, so I'd suggest to put all algorithms under a `algorithms` directory and explain them together under a `## Algorithms` section below.

Reply:
Done